stories/nav-drawer.stories.ts

```diff
@@ -0,0 +1,169 @@
+import { html } from 'lit-html';
+import { ifDefined } from 'lit-html/directives/if-defined';
+import '../igniteui-webcomponents.js';
+import { registerIcon } from '../src/components/icon/icon.registry.js';
+import { Context, Story } from './story';
+import { IgcNavDrawerComponent } from './../src/components/nav-drawer/nav-drawer.js';
+
+// region default
+export default {
+  title: 'Navigation Drawer',
+  component: 'igc-nav-drawer',
+  parameters: {
+    actions: {
+      handles: ['igcOpening', 'igcOpened', 'igcClosing', 'igcClosed'],
+    },
+  },
+  argTypes: {
+    open: {
+      type: 'boolean',
+      description: 'Determines whether the drawer is opened.',
+      control: 'boolean',
+      table: {
+        defaultValue: {
+          summary: false,
+        },
+      },
+    },
+    pinned: {
+      type: 'boolean',
+      description: 'Determines whether the drawer is pinned.',
+      control: 'boolean',
+      table: {
+        defaultValue: {
+          summary: false,
+        },
+      },
+    },
+    disabled: {
+      type: 'boolean',
+      description: 'Determines whether a drawer item is disabled.',
+      control: 'boolean',
+      table: {
+        defaultValue: {
+          summary: false,
+        },
+      },
+    },
+    active: {
+      type: 'boolean',
+      description: 'Determines whether a drawer item is active.',
+      control: 'boolean',
+      table: {
+        defaultValue: {
+          summary: false,
+        },
+      },
+    },
+    position: {
+      type: '"start" | "end" | "top" | "bottom"',
+      options: ['start', 'end', 'top', 'bottom'],
+      control: {
+        type: 'inline-radio',
+      },
+      table: {
+        defaultValue: {
+          summary: 'start',
+        },
+      },
+    },
+  },
+};
+
+interface ArgTypes {
+  content1: string;
+  content2: string;
+  open: boolean;
+  pinned: boolean;
+  disabled: boolean;
+  active: boolean;
+  position: 'start' | 'end' | 'top' | 'bottom';
+}
+
+//end region
```

Comment:
The stories' metadata is nor regenerated on build because of a typo in endregion - it should be one word with an interval after the slashes. 
This comment should be `// endregion`.